src/Handlers/ErrorHandler.php

```diff
@@ -6,50 +6,33 @@
 
 class ErrorHandler extends AbstractHandler
 {
-
     public function register()
     {
         $this->previousHandler = set_error_handler(array($this, 'handle'));
 
         parent::register();
     }
 
-    public function handle()
+    public function handle(...$args)
     {
-        /**
-         * Overloading methods with different parameters is not supported in PHP
-         * through language structures. This hack allows to simulate that.
-         */
-        $args = func_get_args();
+        parent::handle(...$args);
 
-        if (!isset($args[0]) || !isset($args[1])) {
+        if (count($args) < 2) {
             throw new \Exception('No $errno or $errstr to be passed to the error handler.');
         }
 
         $errno = $args[0];
         $errstr = $args[1];
-        $errfile = isset($args[2]) ? $args[2] : null;
-        $errline = isset($args[3]) ? $args[3] : null;
-
-        parent::handle();
-
-        if (!is_null($this->previousHandler)) {
-            $stop_processing = call_user_func(
-                $this->previousHandler,
-                $errno,
-                $errstr,
-                $errfile,
-                $errline
-            );
+        $errfile = $args[2] ?: null;
+        $errline = $args[3] ?: null;
 
+        if ($this->previousHandler) {
+            $stop_processing = ($this->previousHandler)($errno, $errstr, $errfile, $errline);
             if ($stop_processing) {
                 return $stop_processing;
             }
         }
 
-        if (is_null($this->logger())) {
-            return false;
-        }
```

Comment:
This condition is guaranteed to _never_ occur, because the constructor is now strongly typed to only take an argument of `RollbarLogger` type.